<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logowanie</class>
 <widget class="QWidget" name="Logowanie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Zaloguj_Button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zaloguj</string>
   </property>
  </widget>
  <widget class="QLabel" name="Nazwa">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>501</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Text</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Warsztat samochodowy</string>
   </property>
  </widget>
  <widget class="QLabel" name="PESEL">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PESEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="Haslo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Has≈Ço</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Pole_PESEL">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Pole_Haslo">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Login_Info">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Logowanie</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
